mne/io/ctf/ctf.py

```diff
@@ -165,6 +165,35 @@ def _read_segment_file(self, data, idx, fi, start, stop, cals, mult):
                 _mult_cal_one(data_view, this_data, idx, cals, mult)
                 offset += n_read
 
+    def _clean_names(self):
+        """Clean up CTF suffixes from channel names."""
+        for comp in self.info['comps']:
+            for key in ('row_names', 'col_names'):
+                comp['data'][key] = _clean_names(comp['data'][key])
```

Comment:
@OlehKSS this is only cleaning the compensation channels right? We also need to clean the main channels in  ```.info['ch_names'] and .info['chs']```. Essentially, if we clean channel names we should do this consistently for any record of the channel.

Reply:
@dengemann  I've added cleaning of main channels to _clean_names method and deleted overloading of rename_channels method.